test/unit/gateways/realex_test.rb

```diff
@@ -3,7 +3,8 @@
 class RealexTest < Test::Unit::TestCase
   class ActiveMerchant::Billing::RealexGateway
     # For the purposes of testing, lets redefine some protected methods as public.
-    public :build_purchase_or_authorization_request, :build_refund_request, :build_void_request, :build_capture_request
+    public :build_purchase_or_authorization_request, :build_refund_request, :build_void_request,
+      :build_capture_request, :build_otb_request
```

Comment:
@kheang, What if `build_otb_request` were `build_verify_request`? It has the benefit of being more descriptive and following the established pattern.

Reply:
@deedeelavinder I'm happy to change this.  I used "otb" to signal that Realex's request type is "otb", but if that causes confusion, I can swap with verify.  Thoughts?

Reply:
I see now how refund/rebate is handled, consistent with your suggestion. I'll revise!

Reply:
@deedeelavinder Revised per your comments.  Thanks!